beacon_node/beacon_chain/tests/store_tests.rs

```diff
@@ -3050,13 +3050,7 @@ async fn schema_downgrade_to_min_version() {
         )
         .await;
 
-    let min_version = if harness.spec.deneb_fork_epoch.is_some() {
-        // Can't downgrade beyond V18 once Deneb is reached, for simplicity don't test that
-        // at all if Deneb is enabled.
-        SchemaVersion(18)
-    } else {
-        SchemaVersion(16)
-    };
+    let min_version = SchemaVersion(19);
```

Comment:
This should fix the `beacon-chain-tests`. I figured since Electra is still pretty far off, all the downgrades should be checked.

Reply:
Niceee, ty!